juju/juju
# [JUJU-1190] Drop assess_block.py legacy integration test

It is already replicated in the cli test suite.
https://github.com/juju/juju/blob/2.9/tests/suites/cli/block.sh

This PR depends on https://github.com/juju/juju-qa-jenkins/pull/22

## QA steps

Verify that our CI pipeline still runs fine and that https://jenkins.juju.canonical.com/job/test-cli-test-block-commands-lxd/ runs when we expect nw-block-commands to run

## Documentation changes

N/A

## Bug reference

N/A
